modules/engines/engine-core/src/main/resources/META-INF/templates/removeTypeArtifactXSLT.xslt

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Could we remove this file for now as it's not currently used?

Reply:
This could be done in a separate PR as it's a change in the dirigiblelabs module

Reply:
Yes